src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+public class Ball {
+    String ballColor;
+    int ballNumber;
+
+    public Ball() {
+    }
+
+    public Ball(String color, int number) {
+        this.ballColor = color;
+        this.ballNumber = number;
+    }
+
+    public String getBallColor() {
+        return this.ballColor;
+    }
+
+    public void setBallColor(String ballColor) {
```

Comment:
Once the empty constructor will be removed from the code this method will no longer be needed, because all `Ball` instances will always have color set via the constructor.